### ISSUE TITLE
Log Scale broken for variables with 0 or negative values

### ISSUE THREAD

**Issue by alexsielicki** [sandialabs/slycat#960]
Seems that log scales can't contain 0 or smaller values. @pjcross, perhaps we should disable the log radio buttons for any variables that have 0 or smaller values in the Axes Scales UI?

**Comment by pjcross** [comment]
Yes, you can't take the log of zero or negative numbers.  Yes, we should disable that choice for variables whose range includes values of zero or less.

**Comment by alexsielicki** [comment]
@pjcross thanks, I'll do that. 

I will also need to update the validation for axes ranges to prevent 0 or negative values for variables that have a log scale. And update the disabling of log radio buttons each time the axes scales are changed.

**Comment by alexsielicki** [comment]
This is also affected by "Actions > Edit Variable Values" as the user can change the value of an editable variable to be 0 or less.
